Source/WebCore/crypto/SubtleCrypto.cpp

```diff
@@ -153,10 +147,7 @@ static ExceptionOr<std::unique_ptr<CryptoAlgorithmParameters>> normalizeCryptoAl
     case Operations::Decrypt:
         switch (*identifier) {
         case CryptoAlgorithmIdentifier::RSAES_PKCS1_v1_5:
-            if (isRSAESPKCSWebCryptoDeprecated(state))
-                return Exception { ExceptionCode::NotSupportedError, "RSAES-PKCS1-v1_5 support is deprecated"_s };
-            result = makeUnique<CryptoAlgorithmParameters>(params.releaseReturnValue());
-            break;
+            return Exception { ExceptionCode::NotSupportedError, "RSAES-PKCS1-v1_5 support is deprecated"_s };
```

Comment:
Should we even keep this, and remove CryptoAlgorithmIdentifier::RSAES_PKCS1_v1_5 as well?

Reply:
Trying to keep this as this is one way to give an exception similar to Chrome and Firefox if someone tries to use this algorithm.